# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1325535585 18000
#      Mon Jan 02 15:19:45 2012 -0500
# Node ID e530656055aeb356b85f7442d7541a954f558e4c
# Parent  b6d5c60c941cbeabc8cbf98546075c76628190da
doc: add Iain Murry to contributors list

* contributors.in: Add Iain Murray to the list.

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -163,16 +163,17 @@ Christoph Mayer
 Júlio Hoffimann Mendes
 Thorsten Meyer
 Petr Mikulik
 Stefan Monnier
 Antoine Moreau
 Kai P. Mueller
 Hannes Müller
 Victor Munoz
+Iain Murray
 Carmen Navarrete
 Todd Neal
 Philip Nienhuis
 Al Niessner
 Rick Niles
 Takuji Nishimura
 Kai Noda
 Eric Norum
